Frontend/ui-driver/src/Flow.purs

```diff
@@ -266,8 +266,8 @@ getDriverInfoFlow = do
         Nothing -> pure unit
       
       let (Vehicle linkedVehicle) = (fromMaybe dummyVehicleObject getDriverInfoResp.linkedVehicle)
-      void $ pure $ setCleverTapUserProp "Vehicle Variant" linkedVehicle.variant
-
+      void $ pure $ setCleverTapUserProp "V.ehicle Variant" linkedVehicle.variant
```

Comment:
@witcher-shailesh Can you check this log event?

Reply:
Done 👍